cmd/tdex/feeder.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+import (
+	"fmt"
+
+	daemonv2 "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/tdex-daemon/v2"
+	tdexv2 "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/tdex/v2"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	feeder = cli.Command{
+		Name:  "feeder",
+		Usage: "manage price feeds",
+		Subcommands: []*cli.Command{
+			addPriceFeed, startPriceFeed, stopPriceFeed, updatePriceFeed,
+			removePriceFeed, getPriceFeed, listPriceFeeds, listSources,
+		},
+	}
+	addPriceFeed = &cli.Command{
+		Name:   "add",
+		Usage:  "add a new price feed",
+		Action: addPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "base_asset",
+				Usage: "base asset of the market for which the price feed is created",
+			},
+			&cli.StringFlag{
+				Name:  "quote_asset",
+				Usage: "quote asset of the market for which the price feed is created",
+			},
+			&cli.StringFlag{
+				Name:  "source",
+				Usage: "price source to use, e.g. kraken, bitfinex, coinbase etc",
+			},
+			&cli.StringFlag{
+				Name:  "ticker",
+				Usage: "ticker of the market, e.g. XBT/USDT, XBT/EUR etc",
+			},
+		},
+	}
+	startPriceFeed = &cli.Command{
+		Name:   "start",
+		Usage:  "starts price feed",
+		Action: startPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+		},
+	}
+	stopPriceFeed = &cli.Command{
+		Name:   "stop",
+		Usage:  "stops price feed",
+		Action: stopPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+		},
+	}
+	updatePriceFeed = &cli.Command{
+		Name:   "update",
+		Usage:  "update a price feed source and ticker",
+		Action: updatePriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+			&cli.StringFlag{
+				Name:  "source",
+				Usage: "price source to use, e.g. kraken, bitfinex, coinbase etc",
+			},
+			&cli.StringFlag{
+				Name:  "ticker",
+				Usage: "ticker of the market, e.g. XBT/USDT, XBT/EUR etc",
+			},
+		},
+	}
+	removePriceFeed = &cli.Command{
+		Name:   "remove",
+		Usage:  "remove a price feed",
+		Action: removePriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "id",
+				Usage: "id of the price feed to start",
+			},
+		},
+	}
+	getPriceFeed = &cli.Command{
+		Name:   "get",
+		Usage:  "get a price feed",
+		Action: getPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "base_asset",
+				Usage: "base asset of the market for which the price feed is created",
+			},
+			&cli.StringFlag{
+				Name:  "quote_asset",
+				Usage: "quote asset of the market for which the price feed is created",
+			},
+		},
```

Comment:
In spike of the prev comments, these flags can be dropped in favour of a `--id` flag:
```suggestion
			&cli.StringFlag{
				Name:  "id",
				Usage: "id of the price feed to get info about",
				Required: true,
			},
```

Reply:
Done